src/Libraries/Microsoft.Extensions.Http.Resilience/README.md

```diff
@@ -73,6 +73,37 @@ clientBuilder.AddResilienceHandler("myHandler", b =>
 });
 ```
 
+## Known issues
```

Comment:
@IEvangelist Could you, please, review this section? How do you think if I can add it as-is to the following page https://learn.microsoft.com/en-us/dotnet/core/resilience/http-resilience? If not, what should I adjust? Thank you.